Widget Fulfillment Department (#167)
dbos-inc/dbos-demo-apps: widget-fulfillment/README.md

# DBOS Widget Store - Fulfillment Department

Visit the main [widget store](../widget-store/README.md) app for an opportunity to buy a widget!

> After payment is complete, your order comes here for fulfillment (if you have Kafka working, that is, so see the instructions in the Widget Store for setting it up).

Reviewer: ```suggestion
After payment is complete, your order comes here for fulfillment (if you set up Kafka as shown at https://github.com/dbos-inc/dbos-demo-apps/tree/main/widget-store#readme).
```
